main.tf

```diff
@@ -18,5 +18,5 @@ resource "azurerm_template_deployment" "action-group" {
 }
 
 output "action_group_name" {
-  value = "${azurerm_template_deployment.action-group.parameters.actionGroupName}"
+  value = "${var.action_group_name}"
```

Comment:
any reason to do this?

Reply:
```Error: Error running plan: 1 error(s) occurred:

* module.cmc-bulk-print-fail-action-group.output.action_group_name: Resource 'azurerm_template_deployment.action-group' does not have attribute 'parameters.actionGroupName' for variable 'azurerm_template_deployment.action-group.parameters.actionGroupName'```

Reply:
Also just more straightforward and easier to understand. The only downside would be if at some point we mutated it from `var` to `parameters` and the `var` value is no longer the correct one but I think that's negligible.

Reply:
ok